### PR TITLE
✨  Add TV show details page

### PR DESCRIPTION
**Pull request by Dev-dfm**
Add components and styling to TV show details page for issue#4. It also contains the component `NavBar` which is needed for several pages.

### CODE REVIEW

File: src/app/pages/TVShowDetails/TVShowDetails.module.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  font-size: 1.1rem;
+}
+
+.movieDescription {
+  padding: 0 2em;
+  text-align: left;
+  padding-left: 1rem;
+}
+
+footer {
```
Comment (lmachens) [inline src/app/pages/TVShowDetails/TVShowDetails.module.css]: This targets every `footer`.

**Review by lmachens — CHANGES_REQUESTED** [review]
Good job \^o^/
This pull request adds `NavBar` which is part of other pages too. Please add a note about this in the description.

**Review by roabram — APPROVED** [review]
Looks good 👌
BR2

**Review by tmh9x — APPROVED** [review]

**Review by daebert — APPROVED** [review]
Good job!

**Review by SGierszewski — APPROVED** [review]
Good job!! 🥳

**Review by SGierszewski — APPROVED** [review]
Good job!! 🥳

**Review by dhnazarenko — APPROVED** [review]
lgtm 🚀

**Pull request merged**